src/YesSql.Core/Provider/IDbProviderOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using YesSql.Core.Services;
+
+namespace YesSql.Core.Provider
+{
+    public interface IDbProviderOptions
+    {
+        string ProviderName { get; set; }
```

Comment:
I don't really see the value of this property. Hence we could just remove the `IDbPRoviderOptions` class and use `Configuration` directly in the `AddDbProvider` method.